src/frontends/pytorch/src/op/bernoulli.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/random_uniform.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov;
+using namespace ov::op;
+
+OutputVector translate_bernoulli(const NodeContext& context) {
+    num_inputs_check(context, 1, 3);
```

Comment:
could you please provide comment with supported input signatures?

```
  aten::bernoulli(Tensor self, *, Generator? generator=None) -> Tensor:
  aten::bernoulli.out(Tensor self, *, Generator? generator=None, Tensor(a!) out) -> Tensor(a!):
  aten::bernoulli.p(Tensor self, float p, *, Generator? generator=None) -> Tensor:
  aten::bernoulli.Tensor(Tensor self, Tensor p, *, Generator? generator=None) -> Tensor:
  aten::bernoulli.Tensor_out(Tensor self, Tensor p, *, Generator? generator=None, Tensor(a!) out) -> Tensor(a!):
  aten::bernoulli.float_out(Tensor self, float p=0.5, *, Generator? generator=None, Tensor(a!) out) -> Tensor(a!):
```

as I can see operation may have 4 inputs, which case from provided is not supported?

Reply:
Added supported signatures and extend with support of p param